linode/manager
# change: [M3-8468] - Improve action menu styling

## Description 📝
When a user opens the LinodeActionMenu by using their mouse, the first option remain highlighted, even after the user is hovering over a separate option. This can be confusing for our users and negatively impacts accessibility.

It has also been reported that the LinodeActionMenu could use additional visual enhancements during the active state. In particular, when visiting the Linodes landing page, the top of the menu can be difficult to identify for some users because it's aligned with the "Last Backup" column of the Linodes table. This is particularly more evident when a user is using dark mode settings.

## Changes  🔄
- Added an eventHandler to change the `<MenuItem/>` state to focus status `onMouseEnter`
- Changed the active state color for `<MenuItem` while in dark mode

## Target release date 🗓️
N/A

## Preview 📷

- On hovering over menu items

| Before  | After   |
| ------- | ------- |
|![image](https://github.com/user-attachments/assets/f2322663-e2c3-4fcf-b18f-e9650e5cba45)| ![image](https://github.com/user-attachments/assets/9fff9597-6512-4bcf-b07b-b9f3f6a90e08)
|

- Visual enhacement

| Before highlight color change  | After highlight color change   |
| ------- | ------- |
| ![image](https://github.com/user-attachments/assets/f45a8084-e56b-425b-8f1c-aca6d1e3884d)| ![image](https://github.com/user-attachments/assets/13cfc969-5937-481b-8acc-b8f5cf8650c8)|

## How to test 🧪

### Prerequisites
- Open the website in dark theme  

### Verification steps
- Navigate to `localhost:3000/linodes` and click on the `<KebabIcon/>` for any one of the Linodes
- Ensure that the first item('Power Off` in this case) doesn't remain highlighted when you are hovering over other options
- Verify that the `Last Backup` Column of the Linodes table is able to be differentiated from the top of the option menu

## As an Author I have considered 🤔

*Check all that apply*
